iree/samples/vulkan/vulkan_inference_gui.cc

```diff
@@ -723,36 +723,34 @@ int iree::IreeMain(int argc, char** argv) {
         iree_hal_buffer_release(input0_buffer);
         iree_hal_buffer_release(input1_buffer);
         // Marshal input buffer views through a VM variant list.
```

Comment:
Just curious - is this sample working for you [now]? It had issues when I last checked: https://github.com/google/iree/issues/2456 

(API updates are great, just a little worried about bit rot compounding)

Reply:
Untested - should we remove it if we aren't building it?

Reply:
I think it's worth keeping around, just needs some attention to keep it working. I still want to expand on it (e.g. to demo async invocations).

Reply:
SGTM